geonode/favorite/templates/favorite/_favorite.html

```diff
@@ -8,9 +8,9 @@ <h4>{% trans 'Favorite' %}</h4>
     {% if user.is_authenticated %}
       {% if not READ_ONLY_MODE %}
         <div class="clearfix">
-            <button id="favoriteLink" class="btn btn-primary">Add to Favorites</button>
+            <button id="favoriteLink" class="btn btn-primary">{% trans 'Add to Favorites' %}</button>
```

Comment:
Most of the time " are used, some times '
Should we harmonize the quoting?

Reply:
Harmonizing in the GeoNode Translations is tricky. There are some entries in .po files which use " in the strings directly. On the other hand the french language use the ' very often. 
So I had to swap some source code parts with " or ' so that the tags are not interrupted and then error messages are thrown.

Reply:
@afabiani I would approve these changes.